examples/custom-npm-registry/README.md

```diff
@@ -0,0 +1,24 @@
+# Sandpack custom npm registry (proxy)
+
+This project works as a recipe to proxy NPM registries. The main goal is to consume private packages and expose them through a private registry without any authentication token (GitHub or Npm) or requiring any authentication process, 
+
+This new private registry can be used on a Sandpack instance and run private packages public. 
+
+Disclaimer: it would be best if you kept in mind that it will expose all private packages in your account, so be careful where this proxy will be used and the scopes of the authentication token. That's why we initially set this project for read-only purposes, meaning no user would be able to publish changes using this proxy.
```

Comment:
It is possible for users to define only certain packages to expose, if customScopes is `@scope/package-name` instead of `@scope/*` it will only expose that one package, they can even do something like `@scope/design-system*` to expose all packages of the design system or something